content/docs/user-guide/setup-google-drive-remote.md

```diff
@@ -194,19 +194,40 @@ authentication is needed.
 On the first usage of a GDrive [remote](/doc/command-reference/remote), for
 example when trying to `dvc push` for the first time after adding the remote
 with a [valid URL](#url-format), DVC will prompt you to visit a special Google
-authorization web page. There you'll need to sign into your Google account. The
+authentication web page. There you'll need to sign into your Google account. The
 [auth process](https://developers.google.com/drive/api/v2/about-auth) will ask
 you to grant DVC the necessary permissions, and produce a verification code
 needed for DVC to complete the connection. On success, the necessary credentials
 will be saved in a Git-ignored file, located in
-`.dvc/tmp/gdrive-user-credentials.json`.
+`.dvc/tmp/gdrive-user-credentials.json` and they will be used automatically next
+time you run DVC.
```

Comment:
this is not exactly true, they can be used for multiple remotes if you don't specify an option gdrive_user_credentials_file

Reply:
Got it. But "run DVC" doesn't sound exact either. How about
```suggestion
`.dvc/tmp/gdrive-user-credentials.json` by default, and they will be loaded
automatically next time you use this or other GDrive remotes.
```
Maybe even continue this paragraph with the explanation about how to overwrite it, instead of a separate paragraph that repeats the file name.